### PR TITLE
8629 zipfile export format

### PR DESCRIPTION
**Pull request by mvanveen**
Implements zip file export option for jobs (8629).

### CODE REVIEW

File: eventkit_cloud/tasks/task_runners.py [hunk 1]
```diff
@@ -64,6 +64,8 @@ def run_task(self, provider_task_uid=None, user=None, run=None, stage_dir=None,
         logger.debug('Running Job with id: {0}'.format(provider_task_uid))
         # pull the provider_task from the database
         provider_task = ProviderTask.objects.get(uid=provider_task_uid)
+        export_provider = ExportProvider.objects.get(id=provider_task.provider_id)
```
Comment (jsvrcek) [inline eventkit_cloud/tasks/task_runners.py]: ExportProvider here looks like an unused import and database query.

Comment (jj0hns0n) [inline eventkit_cloud/tasks/task_runners.py]: might be a good idea to hook up pylint/pyflakes to catch stuff like this.

Comment (mvanveen) [inline eventkit_cloud/tasks/task_runners.py]: yeah good call.  I need to set myself up with a linter again at least.

Comment (mvanveen) [inline eventkit_cloud/tasks/task_runners.py]: btw thx for flagging Joe!

**Pull request merged**